crypto.go

```diff
@@ -108,6 +110,19 @@ func Ed25519PrivateKeyFromString(s string) (Ed25519PrivateKey, error) {
 	return Ed25519PrivateKey{}, fmt.Errorf("invalid private key with length %v", len(s))
 }
 
+func Ed25519PrivateKeyFromKeystore(ks []byte, passphrase string) (Ed25519PrivateKey, error) {
+	return parseKeystore(ks, passphrase)
+}
+
+func Ed25519PrivateKeyReadKeystore(source io.Reader, passphrase string) (Ed25519PrivateKey, error) {
```

Comment:
This should probably just be `ReadKeystore`. We can extend it to support other types in the future. This would mirror to a generic `PrivateKey.readKeystore` in say Java.

Reply:
@mehcode There isn't a very clean (from the perspective of the sdk user) way to make it generic, my thought was that the inner keystore function would eventually be made more generic and each key type will have the ReadKeystore function.

My concern with that is multi key keystores though -- not sure if we will need those.